iOS/Flows/Home/_New/ConversationViewControllerDataSource+CellRegistration.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MessageCellRegistration.swift
+//  Jibber
+//
+//  Created by Martin Young on 9/22/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+extension ConversationCollectionViewDataSource {
+
+    typealias MessageCellRegistration
+    = UICollectionView.CellRegistration<new_MessageCell,
+                                        (channelID: ChannelId,
+                                         messageID: MessageId,
+                                         dataSource: ConversationCollectionViewDataSource)>
```

Comment:
The datasource is passed into this registration call because we need to call reconfigure on it.